app/src/auth/components/LogoutPage.jsx

```diff
@@ -25,6 +27,7 @@ const LogoutPage = () => {
             })
             if (isMounted()) {
                 dispatch(logoutAction())
+                resetSessionToken()
```

Comment:
Maybe the cleanup can be combined somehow, now added to clear session context here as well